fuuk/people/views.py

```diff
@@ -17,7 +16,7 @@ class ArticleList(ListView):
 
     def get_queryset(self):
         queryset = Article.objects.filter(type__in=('ARTICLE', 'BOOK'))
-        self.years = queryset.values_list('year', flat=True).annotate(Count('year')).order_by('-year')
+        self.years = queryset.values_list('year', flat=True).distinct().order_by('-year')
```

Comment:
`annotate` was used because it is more effective than `distinct` and it should be preferred where needed.